### ISSUE TITLE
I think early-game weapons need to be looked at and rebalanced.

### ISSUE THREAD

**Issue by ampersand55** [CleverRaven/Cataclysm-DDA#32592]
**Is your feature request related to a problem? Please describe.**  
I think currently, early-game (as in pre-forge and no special materials needed) weapons are not very balanced and I would like them to be more balance to see some variety in viable options. In particular, I think spears with reach attack are too good since the health regeneration change and really the only viable melee option one has in the early game pre-armor.

I made a spreadsheet of the early game weapons sorted by my perceived balance.

https://docs.google.com/spreadsheets/d/1E2uPPdBCu22KqVz1nVG3FwoRNQVALSp9TGo5CxwNVi4/edit?usp=sharing

I tried to follow the [page on weapon balance](https://github.com/CleverRaven/Cataclysm-DDA/wiki/Weapons-Balance) as much as I could. The point system for weapons that I devised was:

Damage per 100 moves
+2.5 x +hit value 
+3 for rapid strike 
+1 for high block 
+0.5 for any other technique 
+2 for reach 
+5 for unarmed 
-2 for flimsy

Then for a "balance value" I remove the total skills required to craft the weapon. 

Example:

Forked spear has 6 bash and 17 pierce damage and 116 moves per attack, which is 19.83 damage per 100 moves. 
add +2.5 for +1 in hit (22.33), 
add +1 for high block (23.33), 
add 0.5 for disarm (23.83), 
add +2 for reach attack (25.83), 
remove -2 for flimsy (23.83). 
remove 2 for the requirement of fabrication(2) remove 1 for the requirement of survival(1) (20.83)

20.83 is quite high and I think forked spear should be nerfed.

I used the cudgel as the base balance value (16.71) as I consider it a well balanced weapon, and I suggested buffs for weapons below and nerfs for weapons above that value.

Sorted by balance level:

```
Weapon                 Points Skills   Balance
                                     
Quarterstaff            25.97      4     21.97
Forked Spear            23.83      3     20.83
Copper Spear            23.58      3     20.58
Wooden Spear            23.18      3     20.18
Makeshift crowbar       18.37      0     18.37
Nail knuckles           18.16      0     18.16
Makeshift macuahuitl    21.10      3     18.10
Copper knife            17.72      0     17.72
Homemade Halfpike       22.15      5     17.15
Cudgel                  16.71      0     16.71
2-by-sword              16.68      0     16.68
Cestus                  18.48      3     15.48
Long Pole               20.43      5     15.43
Nail board              15.04      0     15.04
Wooden Tonfa            21.50      7     14.50
Nord                    16.03      2     14.03
Crude sword             15.39      2     13.39
Pair of Steel knuckles  15.53      3     12.53
Peasant Flail           14.92      5      9.92
```

See the spreadsheet for more details.

**Describe the solution you'd like**  
I've made some balance suggestions in the spreadsheet, but those were kind of arbitrary and I'm open for suggestions. The main thing I want is to get the early game weapons looked at.

If we can find some sort of consensus and the senior devs approve, I'd be happy to make a PR.

**Describe alternatives you've considered**  
Some weapons are better than others and part of the skill of the game is to figure out which one's are good. 

It's true that weapons doesn't have to be perfectly balanced. It's fine to have both good and bad weapons. However I think that variety should be encouraged and that there should be a couple of relatively equally good options for situations where it can be justified. 

The crafting options are a selection of what a survivor could consider making, there are no crafting options for useless and pointless things such as cloth swords or food made out of scrap metal. So if there is no real reason to make a particular weapon then it should not be there in first place. There is of-course nothing wrong with suboptimal "for fun" things, but why would someone consider making a bad weapon when they have better options?

Right now with the change to health regeneration, it's frankly stupid to make anything else than a spear with ranged attack for your first melee weapon (unless you play as a martial artist).

I didn't take monster armor and damage type into consideration because 85-90% of the monsters you will be facing in the early game with your first weapon are unarmored ([recent game for reference](https://i.imgur.com/JXHfLNV.png)).

I didn't take strength bash bonus, melee skill bonus or encumbrance movement penalty into consideration. This could be certainly be done to get a more accurate representation of weapon balance, but I think this was enough to get the discussion rolling.

The spreadsheet referenced:

https://docs.google.com/spreadsheets/d/1E2uPPdBCu22KqVz1nVG3FwoRNQVALSp9TGo5CxwNVi4/edit?usp=sharing

**Comment by l29ah** [comment]
Forked spear is junk since it disintegrates after a few zombies.

**Comment by snipercup** [comment]
Don't mind it being more balanced. I always use reach weapons early game and never close-combat weapons. I consider the knife spear nerfed since the more advanced version was implemented and the spike on a stick is the first weapon available with reach. I still craft it to keep the zombies away untill I get the slingshot/bow and go ranged. 

If the spears are more balanced maybe it will be interesting to use close-combat weapons early game. I never use any styles because i dont understand it very well and I wouldn't be surprised if there are others like me that just go hack-n-slash and not invest too much into melee combat. What i mean to say is that i don't support early game weapons have styles as a way to balance things out, that would'nt work for me. on the other hand, maybe it will be a good reason to look into styles.

I don't know how well the spears perform in the default cataclysm because I always play casual with 50% speed and 40% resiliance but a few points taken from the spears should be okay.

**Comment by ampersand55** [comment]
> 
> 
> Forked spear is junk since it disintegrates after a few zombies.

I think it holds up pretty well actually. I got around 110 hits on the debug monster before it turned yellow, and even when yellow it's still better than most other weapons.

It's also very easy to remake when it's damaged as it doesn't require any special materials.

**Comment by l29ah** [comment]
Probably it lasts less if you have more strength or smth. When i was playing it the last time, it disintegrated around 5th regular zombie. It can't be remade in the middle of a zombie run.

**Comment by l29ah** [comment]
I pretty much always use a makeshift crowbar at the early game since the early spears are huge, flimsy and slow, and i need a crowbar anyway for relatively silent looting.

**Comment by ampersand55** [comment]
I used a forked spear in my last two runs and the only time it really wore down was when I accidentally hit something else, such as when I tried to pulp corpses that are already pulped and accidentally hit the ground.

The makeshift crowbar is also around 1.66 points too good in my attempted balance system.

**Comment by glenmack** [comment]
In before Kevin comes in and basically says "weapons aren't equal IRL, live with the unbalanced nature" OR makes you give a meaningful reason for nerfing or buffing each weapon based on it's individual IRL characteristics.

I on the other hand suggest that you rethink the strategy. While balance is an admirable goal, balancing all these weapons would just make looted weapons much better. All of these would suck against a machete, which you can find in a garden shed or the 2nd room of a lab start.

The arbitrary scoring system is a good effort but not robust, nor reflective of a variety of play styles. For example, I have NEVER used any of the top 4 of your best early game weapons, at any stage of play. There's some exception in that I used a later quarterstaff (Ironshod and up) that benefited from a martial art, but that was more in the mid game, after books and forge.

I agree weapons need looking at, but either on a case by case basis, or ALL of them. As a system. Nerfing this particular subset to make them balanced just makes them inferior to all looted weapons, and probably takes a bunch of them out of the game all-together.

**Comment by TychaiosIlithios** [comment]
Well there's a primary question - what major setting the balance of this game has towards weapon balance?
All weapons are equal?
This has pros of varied weapons till endgame and cons - what's the point of looking for a better weapon?
All weapons have features?
I.e. during balancing people try to make weapons that have a usable niche. Pros - when it works. Cons - any switch of balance requires reworking each and every weapon.
All weapons are more or less realistic to their material, but nobody really balances them?
Pros - easier to maintain and focus on other features. Crafting and looting has actual variety regarding better and worse weapons found.
Cons - eventually only few of the dozens of weapons qualify as worthy of the endgame char.

My pick of these options is the last because this game is work in progress and stuff will change many many times, better have ppl focus on other stuff and just mod proper weapon balance for current desired moment should you really want that. Nope not trying to bash your opinion just expressing mine - I got ma knowledge of early weapons buffed by you - thanks a lot!

**Comment by ampersand55** [comment]
As I said in my OP: Weapons doesn't have to be perfectly balanced. It's fine to have both good and bad weapons and some near useless "for fun" weapons. However I think that variety should be encouraged and that there should be a couple of relatively equally good options for situations where it can be justified. 

There are some nudges one can make without affecting realism, and I think some of my suggested balance changes actually improves realism.

For instance, why does a simple wooden spear do more damage per turn than a homemade halfpike? The homemade halfpike is basically a wooden spear with a spike at the end. The adding of a spike shouldn't make it a worse weapon and make it unable to block attacks.

I think higher proficiency in skills should generally make one able to make a better version of similar weapons. I'm thinking the crafting menu consists of things a survivor could consider making, and if there simply is no reason to make a weapon that you just figured out how to make then it doesn't really need to be included in the crafting list. I mean, you probably would know how to make a cloth sausage too, but that isn't included in the crafting menu as there would be no point to it.

An example is the quarterstaff and the peasant flail. Why would becoming better at fabrication make one consider making a peasant flail out of the same materials that they already know how to make a far superior similar weapon out of? 

I suppose I agree that a spear would be the most a logical weapon you first make if you find yourself in a post-apocalyptic scenario without access to metal working. Especially when you're up against lumbering zombies. But that's mainly because of the reach of the spear, not because I would think that a wooden spear would do far more damage than, say, a nail board.

**Comment by ampersand55** [comment]
> I on the other hand suggest that you rethink the strategy. While balance is an admirable goal, balancing all these weapons would just make looted weapons much better. All of these would suck against a machete, which you can find in a garden shed or the 2nd room of a lab start.

A machete is better than all of them already, which is realistic since a purpose build weapon would be better than anything you could cobble together with sticks, stings and scraps.

> 
> The arbitrary scoring system is a good effort but not robust, nor reflective of a variety of play styles. For example, I have NEVER used any of the top 4 of your best early game weapons, at any stage of play. There's some exception in that I used a later quarterstaff (Ironshod and up) that benefited from a martial art, but that was more in the mid game, after books and forge.

I agree it's not very robust, that's why I wanted to get a discussion going. 

I've around a bit with early game weapons and crafting in my experience crafting a forked spear the first day makes the early-game quite easy even without any starting skills.

> 
> I agree weapons need looking at, but either on a case by case basis, or ALL of them. As a system. 

I think the end-game weapons are somewhat balanced already. You could make a case for many of them. 

But the early-game weapons are all very comparable as the characters and the circumstances are generally quite similar at the start of the game.

1. You face similar monsters, mostly zombies where 85-90% have no armor.
2. You generally don't have a martial art to take advantage of special weapons.
3. You generally wear basic clothes with similar encumbrance before you can get tailoring up.
4. You generally don't have access to special tools or materials.

With these variables taken out, comparing melee weapons becomes more clear cut. 

> Nerfing this particular subset to make them balanced just makes them inferior to all looted weapons, and probably takes a bunch of them out of the game all-together.

They're not all nerfs and the purpose was to take them all into the game rather than out.

Finding a baseball bat or a machete or basically any forged weapon on day 1 would still be superior to any weapon I listed.

**Comment by glenmack** [comment]
When I said about looking at weapon balance "ALL of them. As a system". You said,

 >I think the end-game weapons are somewhat balanced already. You could make a case for many of them.

I said all, as a system. then you picked a subset, and said it was balanced, then referred to a different subset. Said it wasn't. The project leads here won't let you cherry pick "sets" or "levels" of content. You're also arguing against someone who agrees, but knows your suggestion as it stands won't fly. You state earlier,

>I think higher proficiency in skills should generally make one able to make a better version of similar weapons.

This is you thinking about weapons as a system. ALL of them. This applies to ALL weapons and is a change I know the people controlling the development would be open to. IF and only IF it is fleshed out and robust. Its not a small change. You'd probably need help and an entire project fork, but some of the old arguments against it are no longer entirely valid.

I feel sure Kevin used to say that weapon sharpening and the kind of improvement this entails wasn't going to be in the game because you can never make a lathe, or a bandsaw, or other high precision stationary shop tools. However, that was before they were in the game passively. Lumber mills have some, Research facilities bottom floors have a plethora of "shop tools". These tools would be necessary for a weapon improvement system.

>For instance, why does a simple wooden spear do more damage per turn than a homemade halfpike? The homemade halfpike is basically a wooden spear with a spike at the end. The adding of a spike shouldn't make it a worse weapon and make it unable to block attacks.

This is a side by side case and a good option for an PR.

I know I said it kind of jokingly, but it's funny because it's true, you're either going to have to case by case all of the early game weapons to acquire the kind of balance you seek, or develop a robust idea for a weapon improvement system, and I am behind you on both those options.

**Comment by ampersand55** [comment]
> This is you thinking about weapons as a system. ALL of them. This applies to ALL weapons and is a change I know the people controlling the development would be open to. IF and only IF it is fleshed out and robust. Its not a small change. You'd probably need help and an entire project fork, but some of the old arguments against it are no longer entirely valid.

Balancing all weapons as a system would indeed be to big of a task for me to handle on my own, but I do not agree with the dichotomy that you either have to do case by case balancing or all weapons at once. While I agree that a robust balance overhaul of all weapons would be preferable, I think this early-weapons subset is distinct enough to balance on its own for the reasons I listed.

**Comment by glenmack** [comment]
>While I agree that a robust balance overhaul of all weapons would be preferable, I think this early-weapons subset is distinct enough to balance on its own for the reasons I listed.

Again, I'm not arguing that that particular subset is balanced, I'm informing you that the idea of balancing these weapons against each other BECAUSE they are early game craft-able weapons isn't going fly with the likes of Kevin.

This game doesn't get balance patches because you think an area is unbalanced, they get balance patches because you prove that IRL the thing is better/worse than something else.

I try and get abstracted hypothetical stuff that is more reasonable for game-play in all the time, and have to spend hours justifying how a single survivor machete update should work for a very slight buff. #32528 

In short balance for balance sake isn't going be allowed.

**Comment by ampersand55** [comment]
Thank you for linking to that PR. I agree with you.

Let me make another argument for why weapons ought to be balanced (not aimed at you specifically). 

First we need to understand how and why a particular weapon is designed. If we look at the historical context and try to understand the choices of the designer who developed a particular weapon we can make educated guesses of their design choices based on their limitations.

1. The limitation of knowledge and skill. A weapon designer can only make weapons they can concessive of, know how to make and how they are used.
2. The limitation of time. A weapon designer only has a finite amount of time to make weapons.
3. The limitation of materials. A weapon designer can only use materials they have access to. Also some materials are a scarce resource, and the weapon designer have to make a choice of how to best use them.
4. The limitation of tools. A weapon designer is limited to the weapons his tools can produce.

Given these limitations, I propose that a weapon designer will invariably produce the best weapon they can. And, if the same weapon could be produced by less knowledge, in less time, using fewer materials and tools, it would have been produced that way instead. 

### Thus, real world weapons are naturally balanced.

Survivors are also a weapon designer, and they will generally follow the same maxim as every other weapon designers: They will produce the best weapon they can under the limitations they are given.

There can of course be weapons variety due to preferences and different conditions, and the conditions in a post-apocalyptic world be different than the conditions where the weapons were originally designed, and probably a bunch of other stuff that need to be accounted for. But I think we should try to use this natural balance as the starting point, and try to simulate these conditions in-game by balancing the weapons by tweaking their in-game equivalents to these limitations.

**Comment by bandti45** [comment]
> Given these limitations, I propose that a weapon designer will invariably produce the best weapon they can. And, if the same weapon could be produced by less knowledge, in less time, using fewer materials and tools, it would have been produced that way instead.

It should be said most historic weapons were made by blacksmiths who did not do the fighting (though I'm not saying that to say they didn't know what they were doing). Also, there were definitely many cases of them lowering practicality a little for aesthetics choices (ornate handles or gold and gem additions).
Plus there are A LOT of real-life factors to take into account (hence why there are so many types of weapons); for instance, how your body is built and your skills are honed and who your fighting. A master fighter can use a large unwieldy weapon to inflict more damage while still being effective in combat.

Not saying this to target anyone just leaving what knowledge I have here!
